portfolio-style.css

```diff
@@ -0,0 +1,258 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+.container {
+    width: 100%;
+    overflow: auto;
+    border: 1px solid #FF0FA8;
+}
+
+.top-bar{
+    overflow: auto;
+    font-family:  'Lucida Console', Monaco, monospace;
+    background-color: #EEC643
+}
+
+.top-stuff  {
+    position: relative;
+}
+
+
+.name {
+    float: left;
+    margin: 0px 10px;
+    font-family: Palatino;
+    color: #EAF2E3;
+    font-size: 1px;
+}
+
+h3 {
+font-size: 26px;
+}
+
+
+.hero{
+    z-index: 10;
+    top: -600px;
+    padding: 0px 20px;
+    color: #ffffff;
+    font-family: Verdana;
+    position: relative;
+    font-size: 32px;
+}
+
+.squire {
+    z-index: 10;
+    top: -640px;
+    padding: 0px 85px;
+    color: #ffffff;
+    font-family: Tahoma, Geneva, sans-serif;
+    position: relative;
+    font-size: 22px;
+
+}
+
+
+.top-links{
+    float: right;
+}
+
+.top-links button {
+    color: #EAF2E3;
+    border: 3px solid #EAF2E3;
+    background-color: Transparent;
+    font-size: 20px;
+    padding: 0px 40px;
+    margin: 0px 100px;
+    border-radius: 3px;
+}
+
+
+/*--Image stuff--*/
+
+.landing-container{
+    width:  100%;
+    height: 800px;
+    background-size: cover
+}
+
+.landing-image {
+    width: 100%;
+    height: 800px;
+}
+
+.sunset-container {
+    float: right;
+    border: 1px solid #040850;
+    overflow: auto;
+    position: relative;
+    height: 800px;
+}
+
+.sunset-image{
+    float: right ;
+    width: 500px;
+    height: 100%;
+}
+
+/*--Logo stuff--*/
+
+.logo-links{
+    margin: 5px 190px;
+}
+
+.instagram-image{
+    width: 220px;
+    height: 220px;
+}
+
+.git-image{
+    width: 200px;
+    height: 200px;
+}
+
+.linkedin-image{
```

Comment:
I would try to find alternative assets that are actually the same size so that you don't need to measure them on the screen